api/pom.xml

```diff
@@ -278,12 +278,19 @@
         <dependency>
             <groupId>jakarta.transaction</groupId>
             <artifactId>jakarta.transaction-api</artifactId>
-            <version>2.0.0</version>
+            <version>2.0.1</version>
         </dependency>
         <dependency>
             <groupId>jakarta.annotation</groupId>
             <artifactId>jakarta.annotation-api</artifactId>
             <version>2.1.0</version>
         </dependency>
+        
+        <dependency>
+            <groupId>jakarta.enterprise</groupId>
+            <artifactId>jakarta.enterprise.cdi-api</artifactId>
+            <version>3.0.1</version>
```

Comment:
Should this be 4.0.0 or is it needed at all? -- I can't recall any usage of CDI from Jakarta Connectors, but it's a large spec so I am probably misremembering.

Reply:
I will look at 4.0.0. It is needed because of the changes to module-info. I'm not specialist in that but it wouldn't compile with the upgrade to Jakarta Transactions 2.0.1 without add CDI as a provided dependency.

Reply:
Thanks for the explanation of where the CDI dependency comes from.  I was only asking out of curiosity and had otherwise approved the review to avoid holding up progress.